sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -26,9 +26,20 @@ affected-services = ["containerd", "kubernetes", "host-containerd", "host-contai
 configuration-files = ["warm-pool-wait-toml"]
 restart-commands = []
 
+[services.pluto]
+configuration-files = ["pluto-toml"]
+restart-commands = []
```

Comment:
This is a little scary in that if we had a restart command here, it could be invoked in an infinite loop since `pluto` generates the same settings that it's affected by.

Is this the first time we've touched a settings generator in this conversion effort? I'm not confident this is the right approach architecturally; settings generators have a more complex producer/consumer relationship with the settings API.

I wonder if the `schnauzer-v2 render ...` machinery is really the better way to go here.